### DISCUSSION TITLE
Image in Paragraph has unexpected behavior

### DISCUSSION THREAD

**Discussion by ObsisMc** [LibrePDF/OpenPDF#770] [General]
**Describe the bug**
Add a string and an image into a Paragraph, the string will overlap with the image.

**To Reproduce**
```
        Document document = new Document(PageSize.A4);
        PdfWriter writer = PdfWriter.getInstance(document, new FileOutputStream("myTest.pdf"));

        Image jpg = Image.getInstance("qv2ray.32.png");
        String test = "test";
        Paragraph paragraph = new Paragraph();
        paragraph.add(test);
        paragraph.add(jpg);

        document.open();
        document.add(paragraph);
        document.close();
```
The image can be any image.

**Expected behavior**
I think maybe they the image should be behind the string.

**Actual behavior**
![图片](https://user-images.githubusercontent.com/71996633/159230762-69166223-bf09-469e-83a8-0aec4f3c766c.png)

**System (please complete the following information):**
 - OS: Ubuntu20.04

**Additional context**
I am not sure whether it is a bug or maybe I can use other methods to get expected behavior?

**Comment by testn** [comment]
Can you try to add `writer.setStrictImageSequence(true);` to see if it solves the problem.

**Comment by ObsisMc** [comment]
@testn It doesn't work wherever I put the code.

**Comment by andreasrosdalw** [comment]
Can you please try different versions of OpenPDF and iText to see if this works correctly in other versions?

**Comment by ObsisMc** [comment]
I want to get result like this ![图片](https://user-images.githubusercontent.com/71996633/159743271-bfa7afcb-1f8d-44e2-ad73-4dacb12d4342.png), example (com.lowagie.examples.objects.images) using `writer.setStrictImageSequence(true)` doesn't put image into Paragraph. I wonder whether it is ok for me to put images into Paragraph.

**Comment by ThreeHmeng** [comment]
Hi, @asturio .I would like to work on this issue! So can I try to fix it? ————from SUS-Lanrand

**Comment by asturio** [comment]
@ObsisMc 
"Fixing" this issue could change the expected behavior. As @andreasrosdal asked, could you compare the functionality using iText?

**Comment by asturio** [comment]
I compared this with iText 5, and the behavior there is the same as with OpenPDF. I think the desired effect can be accomplished  with tables.

**Comment by asturio** [comment]
The best way to add image in flow text is to use a Chunk to wrap that image. Take a look add pdf-toolbox/src/test/java/org/librepdf/openpdf/text/pdf/ImageAfterTextExample.java
